test/src/stores/tracesStore.spec.js

```diff
@@ -121,3 +193,90 @@ describe('TracesSearchStore', () => {
             });
     });
 });
+
+describe('TracesDetailsStore', () => {
```

Comment:
Added tests for the traces detail store, covering both its original functionality and my related traces functionality.